build/deploy.sh

```diff
@@ -10,4 +10,4 @@ fi
 
 # Build the image
 
-make IMAGE_REPOSITORY=$IMAGE_REPOSITORY build
+make IMAGE_REPOSITORY=$IMAGE_REPOSITORY build skopeo-push
```

Comment:
```suggestion
make IMAGE_REPOSITORY=$IMAGE_REPOSITORY build skopeo-push build-catalog-image
```

This also needs to call `build-catalog-image`

@jewzaam The `skopeo-push` target is only for the operator image, not the OLM Catalog. All of the OLM Catalog build/push is taken care of from the `build-catalog-image` make target (`create_catalog_image` in `functions.mk`).